pkg/devfile/adapters/docker/component/adapter_test.go

```diff
@@ -342,102 +341,6 @@ func TestDockerTest(t *testing.T) {
 
 }
 
-func TestDoesComponentExist(t *testing.T) {
```

Comment:
Why are you removing these tests? Could it be possible to adapt them instead?

Reply:
These are docker related tests and we don't support docker anymore. There is a issue regarding the removal of it https://github.com/openshift/odo/issues/4500.